.gitignore

```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
```

Comment:
It looks like merge syntax got here. If we remove these manually, just double check that the .gitignore looks good and we can merge this 👍🏿